src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -114,7 +114,7 @@ public function update( $new_instance, $old_instance ) {
 		$updated_instance['limit']                = $new_instance['limit'];
 		$updated_instance['no_upcoming_events']   = ! empty( $new_instance['no_upcoming_events'] );
 		$updated_instance['featured_events_only'] = ! empty( $new_instance['featured_events_only'] );
-		$updated_instance['jsonld_enable']        = (int) ( ! empty( $new_instance['jsonld_enable'] ) );
+		$updated_instance['jsonld_enable']        = ! empty( $new_instance['jsonld_enable'] );
```

Comment:
Is this to align with V1 or are we deviating from V1 by doing this? I don't know what the handling of this specific setting was for V1, but if we're deviating, will this break backwards compatibility (saving `true` or `false` instead of `1` or `0`)?

Reply:
`true` and `false` will become `1` and `0` when prepared for Db, so this should not make a difference.
Implicit boolean casting or `1` and `0` will, then, do the rest.